lib/gui-utils-tft/src/TFTUtils.cpp

```diff
@@ -1,5 +1,7 @@
 #include "TFTUtils.hpp"
 
+#include "driver/rtc_io.h"
+
```

Comment:
please move the headers, to the main header of this class, to TFTUtils.hpp

Reply:
Done